api/server/runner.go

```diff
@@ -225,6 +225,12 @@ func (s *Server) serve(ctx context.Context, c *gin.Context, appName string, foun
 	default:
 		result, err := runner.RunTask(s.tasks, ctx, cfg)
 		if err != nil {
```

Comment:
We need to log this so the error isn't lost and someone can figure out what's wrong.